target/linux/ipq40xx/image/generic.mk

```diff
@@ -1039,7 +1039,7 @@ define Device/qxwlan_e2600ac-c1
 	DEVICE_VARIANT := C1
 	BOARD_NAME := e2600ac-c1
 	SOC := qcom-ipq4019
-	KERNEL_SIZE := 4096k
+	KERNEL_SIZE := 5120k
```

Comment:
Is there a bootloader limit for the kernel size and is there one?

Cause, I see that FW is just stored in a single partition

Reply:
Also, you forgot to remove `DEFAULT := n` so that after DSA conversion images get built again, I assume it was set due to small kernel size

Reply:
When I compiled, I found that the kernel size exceeded 4096k, resulting in a compilation error and I could not generate img firmware.

Reply:
Sorry I didn't pay attention to this DEFAULT := n

Reply:
That doesn't answer my question, what is the bootloader limit for kernel size, or is there even one?
Cause, if there is none there is no point in defining the kernel limit, and if there is one just set it now and not update every time building breaks.

Reply:
With the default config alone, the kernel size exceeds 4019k.

Must be larger than 4019k to compile the simplest firmware.
So I expanded it to 5120k and there is enough space left.

If you try to compile the e2600ac c1, you will know why I did it

Reply:
no limit

Reply:
Then remove the kernel size completely with that explanation as a dynamic splitter is being used anyway, which avoids the need to change the kernel size completely.

Reply:
Thanks!
ok, i will test it.
If it is fine,i will add a new commit and push it.